policyengine_us/variables/gov/states/ma/tax/income/ma_income_tax_before_refundable_credits.py

```diff
@@ -10,3 +10,4 @@ class ma_income_tax_before_refundable_credits(Variable):
     reference = "https://malegislature.gov/Laws/GeneralLaws/PartI/TitleIX/Chapter62/Section4"
     defined_for = StateCode.MA
     adds = ["ma_income_tax_before_credits"]
```

Comment:
It doesn't matter in this case since [Schedule NTS-L-NR/PY](https://www.mass.gov/doc/2017-schedule-nts-l-nrpy-no-tax-status-and-limited-income-credit/download) already caps the LIC at tax before credits, but this follows [Form 1](https://www.mass.gov/doc/2021-form-1-massachusetts-resident-income-tax-return/download) in checking it's non-negative, and follows the pattern we're looking to replicate across states.
```suggestion

    def formula(tax_unit, period, parameters):
        tax_before_credits = tax_unit("ma_income_tax_before_credits", period)
        non_refundable_credits = tax_unit("ma_non_refundable_credits", period)
        return max_(tax_before_credits - non_refundable_credits, 0)
```